test/onnx/expect/TestOperators.test_convtranspose.expect

```diff
@@ -54,7 +54,7 @@ graph {
     dims: 3
     data_type: 1
     name: "weight"
-    raw_data: "\247\2459\276t\251\032\275\030O\275\275,?R\275#\027.\276\322)\236=Q\2470\276\240\353H\274\322\001\211=\272\332\204\275\336\243\337=@\330\306<n\250\330=~\341\374=\254\375\255\275\n=\217=\024\206\252\275\364\027w=\262\356\315\275\342P\266=(\225\037=<>\232\275\250R\301\275\024\362K=\241\3367>f%\275=\010=\230\274\340\013\031\274\002\006\340=\341\366\010\276\332\007\203=^\231\202\275b\n\344=\331\230\214\275\300\351\033<\316\027\205=yn\007>\020\272\347\274\311\3103>\270\272&\276x\234\014\275(\203D\276\370\231\202<p\262_=\265\214\237\275\322\261\243=\270\222\377\274\036@+\276\331K\027>\312\002\363=rG\307=\3031\035>\256\356\220=\"\217\321=\311\210#>\350\322\036\275\364\261\031\276}\325\301\275pt\322\274\360=\255\274\000\366\252\272\212\336\341=,\337|=\014\266\021\276`\315l\275\010q9\275\252\340\357=\'\374\216\275\263\346\244\275\200z\375\274\264\203H\275\020\026B\275\000\322@\276\375\225/>\362\304\321=/\200\272\275\\\352\365\275\340c\333;\302\221\340=\277l\r\276\250\333\224\275"
```

Comment:
Not sure why this would change. @neginraoof any ideas?

Reply:
Try looking through ONNX release notes.
Try converting to numpy types and checking old vs new.

Reply:
Fixed this. Not sure what the issue was before.